### DISCUSSION TITLE
Daily 24/10

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [grupo-ispc-2023-2-0/workflix-app#272] [Daily]
- ¿Qué hice ayer?
- ¿Qué hice o haré hoy?
- Impedimentos o dificultades

**Comment by MarielaGomez** [comment]
Ayer me puse al día con terminar los cuestionarios y tareas que había en la plataforma, también revisar lo que queda de tarea pendiente con respecto al proyecto, veré si hoy día puedo colaborar para dejar terminadas las que nos queda, hoy tenemos clases a las 18.30hs.

**Comment by BraianTroncoso** [comment]
Ayer segui avanzando con las tareas asignadas, habré realizado cerca de 8, trato de meter mas horas para dejar todo lo más completo posible. Finalicé con las de ocultar, con la de validaciones de la db, y también algunas conexiones, sumado que participé de 3 meets distintas para ayudar a realizar tareas a compañeros. Hoy tengo clases, y voy a seguir realizando las restantes.
Dificultades, las mismas que vengo diciendo en las dailys anteriores, todo sigue igual.
